tests/test_bs.py

```diff
@@ -136,7 +139,7 @@ def _setup_bootstrap_in_namespace(name_space):
                               oracle_server='http://{0}:3030'.format(setup_oracle),
                               genesis_time=GENESIS_TIME.isoformat('T', 'seconds'))
 
-        resp = create_deployment(BOOT_DEPLOYMENT_FILE, name_space, cspec)
+        resp = create_deployment(BOOT_DEPLOYMENT_FILE, name_space, 1, cspec)
```

Comment:
we might want to parameterize that as well. as we scale the network 1 bootstrap node won't be enough.

Reply:
ok